include/stumpless/prival.h

```diff
@@ -0,0 +1,62 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+
+/*
+ * Copyright 2018-2023 Joel E. Anderson
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** @file
+ * Function for converting prival from int to string
+ */
+
+#ifndef __STUMPLESS_PRIVAL_STRING
+#  define __STUMPLESS_PRIVAL_STRING
+
+#  include <stumpless/config.h>
+
+#  ifdef STUMPLESS_SYSLOG_H_COMPATIBLE
+#    include <syslog.h>
+#  endif
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+/**
+ * Gets the string representation of the given prival.
+ *
+ * This is a string literal that should not be modified or freed by the caller.
```

Comment:
The implementation uses `alloc_mem`, which means that this statement is incorrect: the result MUST be freed by the caller to avoid a memory leak.

Reply:
My mistake. I used another file as a template and did not update that info.